mlrun/package/context_handler.py

```diff
@@ -323,3 +305,52 @@ def _collect_mlrun_packagers(self):
                 is_mandatory=False,
                 is_custom_packagers=False,
             )
+
+    def _is_logging_worker(self):
+        """
+        Check if the current worker is the logging worker.
+
+        :return: True if the context belongs to the logging worker and False otherwise.
+        """
+        # If it's a OpenMPI job:
+        if self._context.labels.get("kind", "job") == "mpijob":
+            from mpi4py import MPI
+
+            from mlrun import mlconf
```

Comment:
I tried pip installing `mpi4py` and it failed... So, I don't want it to be mandatory in the requirements as it can lead to trouble, especially when we want to release 1.4. User's that run a mpijob should have mpi4py, if they don't they will get `ModuleNotFoundError` and install it. perhaps we can add validation in the runtime level for when it starts to run it will validate for this package.